docs/podman-create.1.md

```diff
@@ -610,6 +610,13 @@ It will also set the default stop signal to SIGRTMIN+3.
 
 This allow systemd to run in a confined container without any modifications.
 
+Note: On `SELinux` systems, systemd attempts to write to the cgroup
+file system.  Containers writing to the cgroup file system is denied by default.
+You need to turn on the container_manage_cgroup boolean, on SELinux separated
+system, to allow this access.
+
+`setsebool -P container_manage_cgroup 1`
```

Comment:
Probably ought to add setsebool (8) to our list of man pages at the bottom of this and any other page referencing it.

Reply:
Added